src/database/utilities/createRecord.js

```diff
@@ -394,6 +395,36 @@ const createTransactionItem = (database, transaction, item) => {
   return transactionItem;
 };
 
+/**
+ * Create a Message record. This will be sent to the server and requests tables
+ * when an app is upgraded from some version to another.
+ *
+ * @param {Realm}  database    App-wide database interface
+ * @param {String} fromVersion Which version the app is being upgraded from.
+ * @param {String} toVersion   Which version the app is being upgraded too.
+ */
+
+const createUpgradeMessage = (database, fromVersion, toVersion) => {
+  const syncSiteId = database.getSetting(SETTINGS_KEYS.SYNC_SITE_ID);
+
+  const body = {
+    fromVersion: String(fromVersion),
+    toVersion: String(toVersion),
+    syncSiteId: Number(syncSiteId),
+  };
+
+  const message = database.create('Message', {
+    id: generateUUID(),
+    type: 'mobile_upgrade',
+  });
+
+  message.body = body;
```

Comment:
Nit: my personal preference is that I like the use of get accessors, but setters without params can be a bit confusing. Wondering whether a setter with a `type` param would be better (e.g. `message.setBody(type, body)`).  One benefit I can think of of this way is that you could add some validation logic to filter out badly formatted or arbitrary fields that desktop won't understand.